### PR TITLE
LPS-185674 Create the model builder components

### PR DESCRIPTION
**Pull request by igor-franca**
Related [LPS-185674](https://liferay.atlassian.net/browse/LPS-185674)

### CODE REVIEW

**Comment by igor-franca** [comment]
I will wait for the execution of the other tests to send the push adding LPS in the title of the Build Lang commit to correct the `ci:test:sf`

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/VerticalBar/CustomVerticalBar.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	panelWidthMax = 1200,
+	panelWidthMin = 400,
+	position,
+	resize = true,
+	triggerSideBarAnimation,
+	verticalBarItems,
+}: CustomVerticalBarProps) {
+	let verticalBarClassName;
+
+	if (triggerSideBarAnimation) {
```
Comment (naraandrade) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/VerticalBar/CustomVerticalBar.tsx:38]: @igor-franca I believe this is not the updated commit creating this component. I remember we had replaced this if else with creating the `verticalBarClassName` as a template string receiving the position value inside it.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	flex-direction: column;
+	padding: 1rem;
+
+	.navbar-form,
+	.container-fluid {
+		padding: 0 !important;
+	}
+}
+
+.lfr__objects-custom-vertical-bar--left-open {
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss:14]: @igor-franca we should keep a pattern of using either `lfr-objects__` or `lfr__objects-`

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by igor-franca** [comment]
ci:stop

**Comment by igor-franca** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss (hunk 2 above)
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss]: Thanks for this review! Changes have been made :)

**Comment by igor-franca** [comment]
ci:test:object

**Comment by naraandrade** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/2433

**Pull request closed without merging**